code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm

```diff
@@ -0,0 +1,202 @@
+
+/datum/eldritch_knowledge/curse/paralysis
+	name = "Curse of Paralysis"
+	gain_text = "Corrupt their flesh, make them bleed."
+	desc = "Curse someone for 5 minutes of inability to walk. Using a knife, pool of blood, left leg, right leg, a hatchet and an item that the victim touched. "
+	cost = 1
+	required_atoms = list(/obj/item/kitchen/knife,/obj/effect/decal/cleanable/blood,/obj/item/bodypart/l_leg,/obj/item/bodypart/r_leg,/obj/item/hatchet)
+	next_knowledge = list(/datum/eldritch_knowledge/curse/blindness,/datum/eldritch_knowledge/summon/raw_prophet)
+	timer = 5 MINUTES
+
+/datum/eldritch_knowledge/curse/paralysis/curse(mob/living/chosen_mob)
+	. = ..()
+	ADD_TRAIT(chosen_mob,TRAIT_PARALYSIS_L_LEG,MAGIC_TRAIT)
+	ADD_TRAIT(chosen_mob,TRAIT_PARALYSIS_R_LEG,MAGIC_TRAIT)
+
+/datum/eldritch_knowledge/curse/paralysis/uncurse(mob/living/chosen_mob)
+	. = ..()
+	REMOVE_TRAIT(chosen_mob,TRAIT_PARALYSIS_L_LEG,MAGIC_TRAIT)
+	REMOVE_TRAIT(chosen_mob,TRAIT_PARALYSIS_R_LEG,MAGIC_TRAIT)
+
+/datum/eldritch_knowledge/spell/cleave
+	name = "Blood Cleave"
+	gain_text = "At first i didn't know these instruments of war, but the priest told me to use them."
+	desc = "Gives AOE spell that causes heavy bleeding."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/pointed/ash_cleave
+	next_knowledge = list(/datum/eldritch_knowledge/summon/raw_prophet,/datum/eldritch_knowledge/spell/area_conversion)
+
+/datum/eldritch_knowledge/ash_final
+	name = "Ashlord's rite"
+	gain_text = "The forgotten lords have spoken! The lord of ash have come! Fear the fire!"
+	desc = "Bring 3 corpses onto a transmutation rune, you will become immune to fire ,space ,cold and other enviromental hazards. You will gain a spell that passively creates ring of fire around you as well ,as you will gain a powerful abiltiy that let's you breath flames of the beast."
+	required_atoms = list(/mob/living/carbon/human)
+	cost = 3
+	route = "Ash"
+	var/finished = FALSE
+	var/list/trait_list = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOFIRE,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER,TRAIT_BOMBIMMUNE)
+
+/datum/eldritch_knowledge/ash_final/recipe_snowflake_check(list/atoms, loc,list/selected_atoms)
+	if(finished)
+		return FALSE
+	var/counter = 0
+	for(var/mob/living/carbon/human/H in atoms)
+		selected_atoms |= H
+		counter++
+		if(counter == 3)
+			return TRUE
+	return FALSE
+
+/datum/eldritch_knowledge/ash_final/on_finished_recipe(mob/living/user, list/atoms, loc)
+	if(finished)
+		return
+	priority_announce("$^@&#*$^@(#&$(@&#^$&#^@# Fear the blaze, for Ashbringer [user.real_name] has come! $^@&#*$^@(#&$(@&#^$&#^@#","#$^@&#*$^@(#&$(@&#^$&#^@#", 'sound/ai/spanomalies.ogg')
+	user.mind.AddSpell(new /obj/effect/proc_holder/spell/pointed/ash_final)
+	user.mind.AddSpell(new /obj/effect/proc_holder/spell/targeted/fire_sworn)
+	for(var/X in trait_list)
+		ADD_TRAIT(user,X,MAGIC_TRAIT)
```

Comment:
Really recommend using your own trait source.

Reply:
eh, both of these are magical, i dont think anything can remove magical traits